docker/release/fedora24/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM fedora:24
+MAINTAINER Andrew Schwartzmeyer <andschwa@microsoft.com>
+
+# This release Dockerfile uses the CentOS 7 build for the purposes of enabling
```

Comment:
Hm I'm not sure why my name is on this Dockerfile as maintainer, it should be @randomvariable's probably.